miio/airconditioningcompanion.py

```diff
@@ -0,0 +1,99 @@
+from .device import Device
+import enum
+
+
+class OperationMode(enum.Enum):
+    Heating = 0
+    Cooling = 1
+    Auto = 2
+
+
+class FanSpeed(enum.Enum):
+    Low = 0
+    Medium = 1
+    High = 2
+    Auto = 3
+
+
+class AirConditioningCompanionStatus:
+    """Container for status reports of the Xiaomi AC Companion."""
+
+    def __init__(self, data):
+        # Device model: lumi.acpartner.v2
+        #
+        # Response of "get_model_and_state":
+        # ['010500978022222102', '010201190280222221', '2']
```

Comment:
It would be nice to have some sort of description for specific byte indexes (and include a short, informal description of the protocol if possible) already here for readability.